django/contrib/admin/helpers.py

```diff
@@ -214,6 +214,8 @@ def contents(self):
             else:
                 if isinstance(f.remote_field, ManyToManyRel) and value is not None:
                     result_repr = ", ".join(map(six.text_type, value.all()))
+                elif isinstance(f.remote_field, OneToOneField) and value is not None:
```

Comment:
Whatever makes sense. I'm actually think it's not needed but it seems we need a test to verify what happens when the reverse relation is null.